Makefile

```diff
@@ -195,6 +195,16 @@ $(error Invalid target: T=$(T))
 endif
 export T
 
+# Select sound language
+LANGUAGE_SOUND_FOLDER = config/_default_cfg_src_/res/$(LANG)
+ifeq ($(LANG),)
+$(error Please specify the sound language in the command line: LANG=<languageAbbrev>)
+endif
+ifeq ($(wildcard $(LANGUAGE_SOUND_FOLDER)),)
+$(error Invalid target: LANG=$(LANG))
+endif
+export LANG
```

Comment:
It would be better to move the "default to en" logic to the Makefile, rather than the build.sh script. That is because I think that the build should run fine when running `make` directly rather than using the script.

It's also safer to use `$(strip)` around the variable, so that any whitespace at the beginning or end is removed.

Reply:
Done! That does make sense - it also helps `clear.sh` run without the explicit `AUDIO=x` addition as well. 

I'm not sure where you're suggesting to use `$(strip)` - could you clarify that?

Reply:
I'm suggesting to add a line with `AUDIO := $(strip $(AUDIO))` just below `AUDIO ?= en`, so that any potential whitespace is removed. This is already being done for other variables such as the `T` variable.

Reply:
Ah, got it, thanks! Added 171beef